test/fixtures/test-runner/output/coverage-with-mock.snapshot

```diff
@@ -0,0 +1,38 @@
+TAP version 13
+# Subtest: module test with mock
+    # Subtest: tests correct thing
+    ok 1 - tests correct thing
+      ---
+      duration_ms: *
+      type: 'test'
+      ...
+    1..1
+ok 1 - module test with mock
+  ---
+  duration_ms: *
+  type: 'suite'
+  ...
+1..1
+# tests 1
+# suites 1
+# pass 1
+# fail 0
+# cancelled 0
+# skipped 0
+# todo 0
+# duration_ms *
+# start of coverage report
+# ---------------------------------------------------------------------------
+# file                       | line % | branch % | funcs % | uncovered lines
+# ---------------------------------------------------------------------------
+# test                       |        |          |         | 
+#  fixtures                  |        |          |         | 
+#   test-runner              |        |          |         | 
+#    coverage-with-mock      |        |          |         | 
+#     module.js              | 100.00 |   100.00 |  100.00 | 
```

Comment:
I might have got this wrong, but I have only one doubt about this:
we're reporting 100% coverage, but this module is mocked.  
IMHO, I would expect this not to be reported.

Reply:
Thanks for the review.
The test is mocking only sum.js, not module.js.
So I think the correct behavior would be that sum.js is not reported, while module.js is reported.
https://github.com/nodejs/node/pull/59348/files#diff-7affb3bc3a89b147be5c4702e139bc9f0199efb69190fe54b45406c0cb5f3de7R3-R16